web-console/src/druid-models/async-query/async-query.ts

```diff
@@ -37,5 +39,8 @@ export interface AsyncStatusResponse {
       sizeInBytes: number;
     }[];
   };
+  stages?: StageDefinition[];
```

Comment:
does it matter that there seem to be stuff missing from `StageDefinition` that are present in `MSQStagesReport` on the java side? it looks like `shuffle` and `output`, https://github.com/apache/druid/blob/master/extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/report/MSQStagesReport.java#L146

Reply:
I actually added it as part of this PR: https://github.com/apache/druid/pull/16809/files#diff-33ae14291c451661ad8a83ecbca49afc6be1d01a5b882e42725bca19ac22c02fR77

Reply:
hah, oops, idk how i missed that 🙃